interactor.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+import sys
+
+class Interactor:
+
+    def __init__(self, interactor_config=None):
+        self.config = interactor_config
+    
+    def set_config(self, interactor_config):
+        self.config = interactor_config
+    
+    def interact(self, client, generate_save_tag):
+        """
+        Starts a container from a saved image that has already been initialized/indexed.
+
+        The 'interact' hook is used by the image to perform any additional initialization
+        necessary, if any, for the user interaction that follows. Assume that any running
+        process started by the 'init'/'index' hooks is gone.
+        """
+
+        save_tag = generate_save_tag(self.config.tag, self.config.save_id)
+        exists = len(client.images.list(filters={"reference": "{}:{}".format(self.config.repo, save_tag)})) != 0
+        if not exists:
+            sys.exit("Must prepare image first...")
+        
+        interact_args = {
+            "opts": {key: value for (key, value) in map(lambda x: x.split("="), self.config.opts)}
+        }
+
+        print("Starting a container from saved image...")
+        container = client.containers.create("{}:{}".format(self.config.repo, save_tag),
+                                            auto_remove=False, stdin_open=True, tty=True)
```

Comment:
Since we're not passing to `stdin`, why do we need `stdin_open=True` and `tty=True`? If we just use `run` (instead of `create` then `exec_run`) and have `detach=True` and `auto_remove=True` would this be sufficient? See comment below about keeping consistent with existing modules.

Reply:
Just using `run` with `detach=True` and `auto_remove=False` in Python SDK is not sufficient (I have tried `run` with various flags and none works so far). Furthermore, it appears that with `create` in this use case, if you remove `stdin_open=True` you can still have an interactive session, but the container exits if you remove `tty=True` as well.